tests/TestCase.php

```
@@ -28,41 +27,63 @@ abstract class TestCase extends \PHPUnit\Framework\TestCase
     /**
      * Tears down the test environment after each test execution.
      *
      * Closes the application and flushes the logger to ensure a clean state for subsequent tests.
      */
     public function tearDown(): void
     {
+        if ($this->hasOriginalRequestTimeFloat) {
+            $_SERVER['REQUEST_TIME_FLOAT'] = $this->originalRequestTimeFloat;
+        } else {
+            unset($_SERVER['REQUEST_TIME_FLOAT']);
+        }
```

PHPMD (2.15.0)

[error] The method tearDown uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)